CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.22.2] - 2024-07-25
+
+### Fixed
```

Comment:
This feels more like a change than a fix to me. It was working previously, it now needs to be adapted to new Rust build system behavior. No bug nor fix here.